docs/src/1_Mechanics/04_Powers.yaml

```diff
@@ -1,31 +1,31 @@
 Template: # Power name        # Must be unique
-    Name: X                   # Optional. Used for rendering Powers with same name.
-    Type: One of              # Required. Passive, Vulny, Major, Minor, Adversary, Free, House.
-    Category: One or list     # One string or list in sub-cat order
-    Mechanic: Descrip         # Rule/effect. Save info added after. No ending periods.
-    Description: Descrip      # Flavor: 1-2 sentences
+    Name: X # Optional. Used for rendering Powers with same name.
```

Comment:
Was this intentional? It looks like the commit is labeled as `black` formatting, but that should only impact python docs, not yaml. 

All for removing the trailing whitespace elsewhere, but my comment indenting here was intentional

Reply:
It happened automatically, not sure why if it wasn't meant to affect YAML.

Can re-do the spacing in the comments, since I agree it's helpful.

Reply:
Just pushed a commit without saving formatting to fix the comment formatting. Not sure how to adjust my black formatter to not target YAML files but this should work for now.